releases/v2.1.0-beta.20181001.md

```diff
@@ -0,0 +1,84 @@
+---
+title: What&#39;s New in v2.1.0-beta.20181001
+toc: false
+summary: Additions and changes in CockroachDB version v2.1.0-beta.20181001 since version v2.1.0-beta.20180924
+---
+
+## October 1, 2018
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.1.0-beta.20181001.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.1.0-beta.20181001.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.1.0-beta.20181001.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.1.0-beta.20181001.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### SQL language changes
+
+- The entries in the `replicas` column of the `crdb_internal.ranges` virtual table are now always sorted by store
+ID.
+- The `EXPERIMENTAL_RELOCATE` statement no longer temporarily increases the number of replicas in a range more than one above the range's replication factor, preventing rare edge cases of unavailability.
+
+### Command-line changes
+
+- The `--log-dir`, `--log-dir-max-size`, `--log-file-max-size`, and `--log-file-verbosity` flags are now available for the [`cockroach start`](../v2.1/start-a-node.html) and [`cockroach demo`](../v2.1/cockroach-demo.html) commands. Previously, these flags were available for other commands but rarely used or functional. [#30341][#30341] {% comment %}doc{% endcomment %}
```

Comment:
"are now *only* available" (i.e. they're no longer available for anything but `start` and `demo`)

Reply:
Done.